### ISSUE TITLE
Training SSD Inception network with COCO-Text API

### ISSUE THREAD

**Issue by aicaffeinelife** [tensorflow/models#3876]
### System information
- **Top level directory: ~/models/research/object-detection**:
- **A custom code similar to found [here](https://github.com/offbye/tensorflow_object_detection_create_coco_tfrecord/blob/master/README.md)**
- **Linux Ubuntu 16.04**
- **TensorFlow install: From pip wheel**
- **1.5.0**
- **CUDA: 9.0 / CuDNN: 7.0**
- **GPU: Gtx1080(8GB)**
- **Python:3.5.4**

### Description of the problem
I created my tfrecords using the `coco-text` API and since I'm detecting only one class I just set the label to 1. When I run `python object_detection/train.py --logtostderr --pipeline_config_path object_detection/ssd_inception_v2_cocotxt.config --out_dir log_dir` I get the following error: 
```
shape[0] = [1,46] vs. shape[1] = [1,23]
	 [[Node: concat = ConcatV2[N=4, T=DT_FLOAT, Tidx=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ExpandDims, ExpandDims_1, ExpandDims_2, ExpandDims_3, Equal_4/y)]]
INFO:tensorflow:Caught OutOfRangeError. Stopping Training.

```

I have set my `batch_size` to 1 as well. The relevant portion of my code that's creating the tfrecord is: 
```
 bboxes = img_data['bboxes']
    xmins = []
    xmaxs = []
    ymins = []
    ymaxs = []
    
    # the coco format is [left,top,width,height]

    for bbox in bboxes:
        xmins.append(bbox[0])
        xmaxs.append(bbox[0] + bbox[2]) # 
        ymins.append(bbox[1])
        ymins.append(bbox[1] + bbox[3])

    example =  tf.train.Example(features=tf.train.Features(feature={
        'image/height':dataset_util.int64_feature(img_data['height']),
        'image/width':dataset_util.int64_feature(img_data['width']),
        'image/object/bbox/xmin':dataset_util.float_list_feature(xmins),
        'image/object/bbox/xmax':dataset_util.float_list_feature(xmaxs),
        'image/object/bbox/ymin':dataset_util.float_list_feature(ymins),
        'image/object/bbox/ymax':dataset_util.float_list_feature(ymaxs),
        'image/object/class/label':dataset_util.int64_list_feature(img_data['labels']),
        'image/object/class/text':dataset_util.bytes_list_feature(img_data['text']),
        'image/encoded':dataset_util.bytes_feature(img_data['data']),
        'image/format':dataset_util.bytes_feature('jpeg'.encode('utf-8')) 

    }))

    return example

```
Another interesting aspect is that `ConcatOp` fails with size mismatch of different sizes each time (size[0] and size[1]) are different each time. I am at a loss to where to localize this error to even begin debugging. So any help will be appreciated.

**Comment by nguyeho7** [comment]
Are you sure you have to sum bbox[0] and bbox[2] for xmaxs? isn't just bbox[2] enough?

**Comment by aicaffeinelife** [comment]
So from what I have read, COCO annotations have the form [top,left, width, height]. Based on that I derive the xmax and the ymax

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
